site/mkdocs.yml

```diff
@@ -59,7 +59,7 @@ nav:
     - Spark SQL: spark.md
     - Spark Streaming: spark-structured-streaming.md
     - Time Travel: spark#time-travel
-  - Presto: https://prestosql.io/docs/current/connector/iceberg.html
+  - Trino (formerly PrestoSQL): https://trino.io/docs/current/connector/iceberg.html
```

Comment:
This title is too long for the header. Can you remove "(formerly PrestoSQL)"?

Reply:
Updated